README.md

```diff
@@ -1,20 +1,41 @@
 # HERO Software Developement Kit
-
-## About
-The HERO SDK contains the following packages:
-* PULP HERO GCC Toolchain with RISC-V offloading support through OpenMP 4.5.
-* PULP HERO Linux driver and `libpulp` runtime library for offloading support.
-* Host Linux kernel and Buildroot root file system.
-* PULP SDK ([link](https://github.com/pulp-platform/pulp-sdk));
-* HERO Examples using OpenMP Heterogeneous Accelerator Execution Model.
-
-### Prerequisites (on Ubuntu 16.04)
+**HERO** \[1\] combines a **PULP-based** \[2\] open-source parallel manycore accelerator implemented on FPGA with a hard ARM Cortex-A multicore host processor running full-stack Linux. HERO is the **first heterogeneous system architecture** that mix a powerful ARM multicore host with a highly parallel and scalable many-core accelerator based on a **RISC-V cores**.
```

Comment:
How about directly linking the papers? For example:
```markdown
[HERO][hero-paper] combines ...
```
and at the end of the file
```markdown
[hero-paper]: https://arxiv.org/abs/1712.06497
```
If you do not want to do that, I suggest to put the references section to the end of the file.